example/client/include/KVMessageGenerator.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <chrono>
+#include <map>
+#include <set>
+#include <vector>
+#include <cstdio>
+#include <cstring>
+#include <sys/param.h>
+#include <unistd.h>
+#include <yaml-cpp/yaml.h>
+#include <TimeUtils.hpp>
+
+#include "Logger.hpp"
+#include "kv_replica_msgs.cmf.hpp"
+#include "messages/ClientRequestMsg.hpp"
+#include "KVBCInterfaces.h"
```

Comment:
please cleanup the header files. Remove the ones not needed and also move those to .cpp which are used only there.

Reply:
fixed